lib/plugins/aws/package/compile/events/cloudWatchEvent/index.js

```diff
@@ -11,9 +11,36 @@ class AwsCompileCloudWatchEventEvents {
       'package:compileEvents': this.compileCloudWatchEventEvents.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8026
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'cloudwatchEvent', {
       type: 'object',
+      properties: {
+        event: { type: 'object' },
+        input: {
+          oneOf: [{ type: 'string', maxLength: 8192 }, { type: 'object' }],
+        },
+        inputPath: { type: 'string', minLength: 1, maxLength: 256 },
+        inputTransformer: {
+          type: 'object',
+          properties: {
+            inputPathsMap: {
+              type: 'object',
+              additionalProperties: { type: 'string', minLength: 1 },
+            },
+            inputTemplate: { type: 'string', minLength: 1, maxLength: 8192 },
+          },
+          required: ['inputTemplate'],
+          additionalProperties: false,
+        },
+        description: { type: 'string', maxLength: 512 },
+        name: { type: 'string', pattern: '[a-zA-Z0-9-_.]+', minLength: 1, maxLength: 64 },
+        enabled: { type: 'boolean' },
+      },
+      oneOf: [
+        { required: ['input'] },
+        { required: ['inputPath'] },
+        { required: ['inputTransformer'] },
```

Comment:
I think we should also add `required: []`, as technically we do not require any of this options